content/articles/2019-05-27-Bonding-period-ends.md

```diff
@@ -0,0 +1,82 @@
+Title: Coding phase starts!
+Date: 2019-05-27 11:30
+Category: GSCOC
+Tags: GSOC, GSOC19, poliastro
+slug: 2019-05-27-poliastro-gsoc-jorge
+lang: en
+Author: Jorge Martínez
+
+
+## The End of the bonding period
+
+Today bonding period ends for all students selected at GSoC'19. The aim of this
+period was to learn more about your project community: get in touch with
+mentors, other developers and their future ideas on the software...
+
+As I said in my last post, I started contributing to poliastro several months
+ago, even before GSOC was announced. Therefore, and since I am taking my final
+exams, all my efforts were towards passing them.
+
+But today the coding phase starts! That means we all should start working on
+our proposal features.
+
+## The coding phase begins!
```

Comment:
I think we can get rid of this section! More or less everybody knows, and it's very generic :)